flake.nix

```diff
@@ -77,14 +79,17 @@
               nonCargoBuildFiles = path: _type: builtins.match ".*(json|sql|wasm.gz)$" path != null;
               includeFilesFilter = path: type:
                 (craneLib.filterCargoSources path type) || (nonCargoBuildFiles path type);
+
+              # Crane doesn't know which version to select from a workspace, so we tell it where to look
+              crateInfo = craneLib.crateNameFromCargoToml { cargoToml = inputs.holochain + "/crates/holochain/Cargo.toml"; };
             in
             craneLib.buildPackage {
               pname = "holochain";
-              version = "workspace";
```

Comment:
I'd done this in the old Holonix for some unknown reason, I don't think it's something we want to keep until a reason appears